### PR TITLE
Grammar correction in en_US.json

### PR DESCRIPTION
**Pull request by USBDongleGoblin**
Corrected/nitpicked grammar and improved readability in various strings.

### CODE REVIEW

**Comment by KosmX** [comment]
Thanks for language correction.

Until you keep adding changes, the PR should be draft.
If you think you're ready, press ready-for-review.

File: emotesMain/src/main/resources/assets/emotecraft/lang/en_US.json [hunk 1]
```diff
@@ -1,68 +1,68 @@
...
-  "emotecraft.otherconfig.yratio": "Y axis moving influence in stop threshold",
-  "emotecraft.otherconfig.yratio.tooltip": "bigger value means bigger influence",
-  "emotecraft.otherconfig.loadbuiltin": "Load built-in emotes",
+  "emotecraft.otherconfig.validate": "Validate Emotes",
+  "emotecraft.otherconfig.validate.tooltip": "Mainly intended for PvP worlds. Disallows emotes that offset the player too much.",
+  "emotecraft.otherconfig.validationThreshold": "Validation Threshold",
+  "emotecraft.otherconfig.validationThreshold.tooltip": "How much offset is too much.",
+  "emotecraft.otherconfig.dark": "Use Dark Emote Wheel",
+  "emotecraft.otherconfig.showicon": "Show Icons Only (if available)",
+  "emotecraft.otherconfig.showicon.tooltip": "REMOVE THIS STRING? IDK",
```
Comment (KosmX) [inline emotesMain/src/main/resources/assets/emotecraft/lang/en_US.json]: Just remove it!

https://github.com/KosmX/emotes/blob/c344d18d7f9c8f04a7b76016c0644540a94b893a/emotesMain/src/main/java/io/github/kosmx/emotes/main/config/ClientConfig.java#L16

And change the second bool to `false` to disable tooltip for this entry.

Comment (USBDongleGoblin) [reply emotesMain/src/main/resources/assets/emotecraft/lang/en_US.json]: Done! Sorry, I'm not completely familiar with this code base, and Minecraft modding in general.

Comment (KosmX) [reply emotesMain/src/main/resources/assets/emotecraft/lang/en_US.json]: No problem.
Just what if I don't read the changes and I just merge it with that string.

Comment (USBDongleGoblin) [reply emotesMain/src/main/resources/assets/emotecraft/lang/en_US.json]: No need, I fixed it myself [794a03f](https://github.com/KosmX/emotes/pull/254/commits/794a03fa6d164d1598884538eeac35fd4b3496b5) :)

**Comment by USBDongleGoblin** [comment]
Should be ready to merge now.

**Comment by USBDongleGoblin** [comment]
Just added some punctuation to the end of some strings and changed one.

Really, I'm done now.

**Review by KosmX — APPROVED** [review]
Looks good, thank you!

**Pull request merged**